mirrorlist/mirrorlist_statistics.py

```diff
@@ -21,18 +21,18 @@
 # LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
 
-import sys, pylab, time, getopt
+import sys
+import pylab
+import time
+import getopt
 
 start = time.clock()
 
 logfile = None
 dest = None
 offset = 0
+embargoed_countries = ['CU', 'IR', 'KP', 'SD', 'SY']
```

Comment:
The configuration file might be a good compromise, especially considering the config file could be trimmed down to only contain the information of interest.
It's just that from experience it's easier to adjust a configuration file than doing a new release or using the hotfix mechanism when we need to adjust this kinds of thing.

Note: do we store in the DB the country embargoed? I'm not even sure it's there.